python/ct/serialization/tls_message_test.py

```diff
@@ -209,6 +222,25 @@ def test_encode_repeated_too_short_fails(self):
         test_message.ClearField("vector_uint32")
         self.verify_encode_fails(test_message)
 
+    def test_encode_repeated_fixed_length_too_short_fails(self):
+        test_message = test_message_pb2.TestMessage()
+        test_message.CopyFrom(valid_test_message)
+        test_message.ClearField("vector_fixed_bytes")
+        self.verify_encode_fails(test_message)
+
+    def test_encode_repeated_fixed_length_wrong_length_fails(self):
+        test_message = test_message_pb2.TestMessage()
+        test_message.CopyFrom(valid_test_message)
+        test_message.vector_fixed_bytes.append("\xab\x00")
+        self.verify_encode_fails(test_message)
+
+    def test_encode_repeated_fixed_length_too_long_fails(self):
```

Comment:
using both 'fixed_length' and 'too_short'/'too_long' is a bit confusing.    for the former, fixed_length, you mean the 4 byte length.  For the latter you mean how many repetitions can be permitted.

how about instead naming these:
 test_encode_repeated_fixed_length_too_few_fails 
and
 test_encode_repeated_fixed_length_too_many_fails